Review section for Chart Control. (#609)
telerik/maui-docs: controls/chart/series/financial/financial-indicators.md

title: Financial Indicators
page_title: .NET MAUI Chart Documentation - Financial Indicators
slug: chart-series-financial-indicators
description: Learn more about the financial indicators that are used for keeping track of stock prices and patterns of price changes over time.
position: 2
---

# .NET MAUI Chart Financial Indicators

The financial, or also called stock, indicators are used for keeping track of stock prices and patterns of price changes over time. For more information about what indicators are and what their purpose is, refer to the [Short information about financial indicators](https://www.investopedia.com/terms/t/technicalindicator.asp) article.

> With using the indicators in the `RadCartesianChart`, you need to add them in the same way you add any other Cartesian series. Every indicator has a related formula by which it calculates the expected result. All you need to do is provide the needed data.

Reviewer: ```suggestion
When using the indicators in the `RadCartesianChart`, you need to add them in the same way you add any other Cartesian series. Every indicator has a related formula by which it calculates the expected result. All you need to do is provide the needed data.
```

> With setting their properties and getting them ready for displaying your stock data, the financial indicators are two types:

Reviewer: ```suggestion
When setting their properties and getting them ready for displaying your stock data, consider the following financial indicators types:
```
